frameworks/Java/spring/src/main/java/hello/web/JsonHandler.java

```diff
@@ -0,0 +1,40 @@
+package hello.web;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.function.HandlerFunction;
+import org.springframework.web.servlet.function.ServerRequest;
+import org.springframework.web.servlet.function.ServerResponse;
+
+@Component
+public class JsonHandler implements HandlerFunction<ServerResponse> {
+
+	private static final Map<String, String> MESSAGE = Map.of("message", "Hello, world!");
```

Comment:
Per the rules of this test, the object needs to be instantiated on each request. This optimization is not allowed.

Reply:
Sorry I missed that, I will update the PR accordingly.